server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -271,11 +271,13 @@ public void run() {
         while (!nextLogMaker.isShutdown()) {
           DfsLogger alog = null;
           try {
-            log.debug("Creating next WAL");
+            if (log.isDebugEnabled())
```

Comment:
Seem like this check should be removed.

Reply:
Yeah thanks.  I missed the slf4j in this class.